package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "death-valey",
+  "version": "1.0.0",
+  "description": "gulp package",
+  "main": "index.html",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
please, add some scripts like to use your project for local development, build and etc, in other case this file is almost meaningless 😄